### DISCUSSION TITLE
include with .update() ?

### DISCUSSION THREAD

**Discussion by githubUser982** [sequelize/sequelize#16228] [Help & Questions] [answered]
Hello. I am using a query with a required include in order to find rows that have an associated entry.

```
let query = {
	include: { model: marketplacecards, required: true }
}

let cards = await usercards.findAll(query) 
let count = await usercards.count(query)

await usercards.update({ folder: "bob" }, query) // updates EVERY row, the include is ignored
```

However, for update(), the include has no effect. Is this intended/normal?
Is there some workaround other than fetching all the rows? (thousands of times slower)

**Comment by ephys** [comment, marked as answer]
Yes, there is no `include` option in the `update` method

**Reply by githubUser982** [reply]
So it would seem. It would be good if it was there, like the other methods.
